Features/ManageabilityPkg/Library/ManageabilityTransportSerialLib/Dxe/ManageabilityTransportSerial.c

```diff
@@ -0,0 +1,377 @@
+/** @file
+
+  Serial instance of Manageability Transport Library
+
+  Copyright (c) 2024, ARM Limited. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+*/
+
+#include <Uefi.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/SerialPortLib.h>
+#include <Library/DebugLib.h>
+#include <Library/MemoryAllocationLib.h>
+#include <Library/ManageabilityTransportLib.h>
+#include <Library/ManageabilityTransportIpmiLib.h>
+#include <Library/ManageabilityTransportHelperLib.h>
+
+#include "ManageabilityTransportSerial.h"
+
+MANAGEABILITY_TRANSPORT_SERIAL  *mSingleSessionToken = NULL;
+
+EFI_GUID  *SupportedManageabilityProtocol[] = {
+  &gManageabilityProtocolIpmiGuid,
+};
+
+UINT8  NumberOfSupportedProtocol = (sizeof (SupportedManageabilityProtocol)/sizeof (EFI_GUID *));

```

Comment:
file global variable should start with small letter 'm'.

Reply:
Acked.